Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -223,7 +223,12 @@ static void ThrottleCallback(u64 last_time, s64 cyclesLate)
 // SystemTimers::Init
 void PreInit()
 {
-  if (SConfig::GetInstance().bWii)
+  ChangePPCClock(SConfig::GetInstance().bWii ? Mode::Wii : Mode::GC);
+}
+
+void ChangePPCClock(Mode mode)
```

Comment:
Should we adjust the ticks of currently scheduled events when changing clock in the middle of emulation?